pkg/sql/logictest/testdata/logic_test/alter_column_type

```diff
@@ -523,3 +523,9 @@ SELECT x FROM t28 ORDER BY x
 5
 10
 15
+
+# Regression 50277, ensure ColumnConversionValidate type conversion doesn't
+# error before running the online schema change.
+statement ok
+CREATE TABLE t29 (x INT8);
```

Comment:
Can you add some data in the table, and a case that errors out (if it should)

Reply:
Done both.